idaes/generic_models/properties/core/eos/tests/test_ideal.py

```diff
@@ -226,8 +226,9 @@ def test_dens_mol_phase_liq(m):
         m.params.get_component(j).config.dens_mol_liq_comp = dummy_call
 
     assert str(Ideal.dens_mol_phase(m.props[1], "Liq")) == str(
-        sum(m.props[1].mole_frac_phase_comp["Liq", j]*42
-            for j in m.params.component_list))
+                1/(1/42*m.props[1].mole_frac_phase_comp["Liq", "a"] +
```

Comment:
Just curious why this revision was reformulated in this way.

Reply:
That is what the expression actually looks like now: `rho = 1/(sum x_i*1/v_mol_i for all i)`